authentication/understanding-identity-provider.adoc

```diff
@@ -72,6 +72,12 @@ link:http://openid.net/specs/openid-connect-core-1_0.html#CodeFlowAuth[Authoriza
 Once an identity provider has been defined, you can
 xref:../authentication/using-rbac.adoc#authorization-overview_using-rbac[use RBAC to define and apply permissions].
 
+[NOTE]
+====
+When configuring multiple identity providers, some of them, such as `Request header` and `HTPasswd` can not be
+configured together.
```

Comment:
linguistic nit: instead of "can not be configured together" maybe "are mutually exclusive"?

summoning @bergerhoffer for language advise :pray: :)

Reply:
Rephrased this section. Please review it again.